SerialPrograms/Source/PokemonSV/Inference/Battles/PokemonSV_NormalBattleMenus.h

```diff
@@ -88,6 +88,27 @@ class TerastallizingDetector : public StaticScreenDetector{
 
 
 
+class SwapMenuDetector : public StaticScreenDetector {
+public:
+    SwapMenuDetector(Color color);
+
+    virtual void make_overlays(VideoOverlaySet& items) const override;
+    virtual bool detect(const ImageViewRGB32& screen) const override;
+
+    //  Returns -1 if not found.

```

Comment:
If that commented code was taken from somewhere else, it should be removed!

Reply:
Removed. I was originally going to implement them as well but realized it wasn't needed for this program.